go/cmd/dolt/commands/tblcmds/import.go

```diff
@@ -291,7 +293,7 @@ func executeMove(dEnv *env.DoltEnv, force bool, mvOpts *mvdata.MoveOptions) int
 			return 1
 		}
 
-		if srcFileLoc.Format == mvdata.JsonFile && mvOpts.SchFile == "" {
+		if srcFileLoc.Format == mvdata.JsonFile && mvOpts.Operation != mvdata.UpdateOp && mvOpts.SchFile == "" {
```

Comment:
I think it's easier to read == positiveCase rather than != negativeCase.

```suggestion
		if srcFileLoc.Format == mvdata.JsonFile && mvOpts.Operation == mvdata.OverwriteOp && mvOpts.SchFile == "" {
```

Reply:
I didn't know you could suggest code changes! GitHub has so many features that I don't know about...but I removed the CSV test and this seems like a good change to make so I've committed it as well. I'll merge as soon as Jenkins passes!